CHANGELOG.md

```diff
@@ -259,6 +259,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   Please [see the documentation](/docs/config.md#json-schema-support) for more
   on this.
 
+* The new `jj sign` command allows signing commits.
```

Comment:
nit: you'll need to move this to the unreleased section.

Reply:
Ah thanks. Another sign that I need to get this PR done. 😅